core/src/main/java/com/sequenceiq/distrox/v1/distrox/fedramp/FedRAMPModificationService.java

```diff
@@ -0,0 +1,46 @@
+package com.sequenceiq.distrox.v1.distrox.fedramp;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.auth.altus.EntitlementService;
+import com.sequenceiq.cloudbreak.common.gov.CommonGovService;
+import com.sequenceiq.cloudbreak.common.provider.ProviderPreferencesService;
+import com.sequenceiq.distrox.api.v1.distrox.model.DistroXV1Request;
+import com.sequenceiq.distrox.api.v1.distrox.model.database.DistroXDatabaseAvailabilityType;
+import com.sequenceiq.distrox.api.v1.distrox.model.database.DistroXDatabaseRequest;
+
+@Service
+public class FedRAMPModificationService {
```

Comment:
The naming should be `FedRampMofificationService` (i.e., not using full caps even within abbreviations)
Or `Fedramp...`, just make sure to be consistent with Entitlement service's check method name